### PR TITLE
SB-29456 - Using config parameter for request timeout

### PR DESCRIPTION
**Pull request by karthik-tarento**
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/SB-29456" title="SB-29456" target="_blank">StorySB-29456</a>  Need configuration parameter for request timeout value
  </td></table>
  <br />
 

Creating a PR for JIRA Story - SB-29456

### CODE REVIEW

**Comment by karthik-tarento** [comment]
@AmiableAnil - Can you please review this PR?

File: content-api/content-service/app/controllers/BaseController.scala [hunk 1]
```diff
@@ -26,6 +26,7 @@ abstract class BaseController(protected val cc: ControllerComponents)(implicit e
         new util.HashMap[String, AnyRef]()).asInstanceOf[java.util.Map[String, AnyRef]]
     val mimeTypesToCheck = List("application/vnd.ekstep.h5p-archive", "application/vnd.ekstep.html-archive", "application/vnd.android.package-archive",
         "video/webm", "video/x-youtube", "video/mp4")
+    val actorTimeout: Long = Platform.getLong("actor.timeoutMillisec", 120000L)
```
Comment (AmiableAnil) [inline content-api/content-service/app/controllers/BaseController.scala:29]: Keep the default value 30000L. In your env, you can configure it 120000L in config file.

File: content-api/content-service/conf/application.conf [hunk 2]
```diff
@@ -736,3 +736,6 @@ collection {
 		}
 	}
 }
+
+#Request timeout
+actor.timeoutMillisec = 120000
```
Comment (AmiableAnil) [inline content-api/content-service/conf/application.conf:741]: Keep the default value 30000L. In your env, you can configure it 120000L in config file.

**Comment by karthik-tarento** [comment]
@AmiableAnil -- updated the value and now using 30 sec as default value.

**Pull request merged**